### DISCUSSION TITLE
Replace Email Subject

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1414] [Feature Requests]
This is useful if you are using encryption.The email subject cannot be encrypted as it is one of the headers. To prevent revealing the contents of emails I think that you should provide feature that will replace the subject with name of website or something randomly generated.

**Comment by 4b15ed24f576** [comment]
Hey, you're right, email subject can not be encrypted. 
Though it's not randomly generated, we have a feature to automatically replace the subject of PGP encrypted emails with a generic subject.

When you set up PGP encryption for your mailbox, SimpleLogin replace by default the email subject by "Encrypted Email". However, you can actually set a custom generic subject when your alias use PGP encryption. 

To do so, go to [Mailboxes](https://app.simplelogin.io/dashboard/mailbox/) > **Select the mailbox with PGP enabled** > Go to the section **"Hide email subject when PGP is enabled"**.

<img width="1217" alt="iBt77Pb9NCfJUk5u" src="https://user-images.githubusercontent.com/116150138/201381488-d64ba59a-dbf5-400c-9da1-3de7b1f691a2.png">
